context/app/static/js/components/Detail/FileBrowserDUA/utils.js

```diff
@@ -1,5 +1,5 @@
-function getDUAText(data_access_level) {
-  if (data_access_level === 'protected') {
+function getDUAText(mapped_data_access_level) {
+  if (mapped_data_access_level === 'protected') {
```

Comment:
If the values are now capitalized we need to change `protected` to `Protected`.

Reply:
Sorry! Yes!